ui/app/AppLayouts/Chat/controls/community/HoldingsDropdown.qml

```diff
@@ -38,275 +39,263 @@ StatusDropdown {
+                    PropertyChanges {target: holdingsTabs; sourceComponent: ensLayout; addButtonEnabled: d.ensReady}
                 }
             ]
-            StatusIconTextButton {
-                visible: root.withOperatorSelector
-                Layout.leftMargin: 16
-                Layout.topMargin: 12
-                spacing: 0
-                statusIcon: "next"
-                icon.width: 12
-                icon.height: 12
-                iconRotation: 180
-                text: qsTr("Back")
-                onClicked: loader.sourceComponent = operatorsSelectorView
-            }
-            StatusSwitchTabBar {
-                id: tabBar
-                Layout.preferredWidth: 273 // by design
-                Layout.margins: 8
-                Layout.topMargin: root.withOperatorSelector ? 12 : 16
-                Layout.preferredHeight: 36 // by design
-                currentIndex: d.currentTabIndex
-                onCurrentIndexChanged: d.currentTabIndex = currentIndex
-                StatusSwitchTabButton {
-                    text: qsTr("Token")
-                    fontPixelSize: 13
-                }
-                StatusSwitchTabButton {
-                    text: qsTr("Collectible")
-                    fontPixelSize: 13
-                }
-                StatusSwitchTabButton {
-                    text: qsTr("ENS")
-                    fontPixelSize: 13
-                    enabled: false // TODO
+
+            tabLabels: [qsTr("Token"), qsTr("Collectible"), qsTr("ENS")]
+            state: [d.tokensState, d.collectiblesState, d.ensState][currentIndex]
+
+            currentIndex: d.currentTabIndex
+            onCurrentIndexChanged: d.currentTabIndex = currentIndex
+
+            onAddClicked: d.addClicked()
+
+            Connections {
+                target: backButton
+
+                function onClicked() {
+                    content.state = d.operatorsState
                 }
             }
-            Loader {
-                id: tabsLoader
-                Layout.fillWidth: true
-                Layout.margins: 8
-            }
         }
     }
 
     Component {
-        id: tokensCollectiblesLayout
+        id: tokensLayout
 
-        ColumnLayout {
-            spacing: 0
-            StatusPickerButton {
-                Layout.fillWidth: true
-                Layout.preferredHeight: 36
-                bgColor: Theme.palette.baseColor5
-                contentColor: Theme.palette.directColor1
-                text: d.isTokensLayout ? root.tokenName : root.collectibleName
-                font.pixelSize: 13
-                image.source: d.isTokensLayout ? root.tokenImage : root.collectibleImage
-                onClicked: loader.sourceComponent = extendedView
+        TokensPanel {
+            id: tokensPanel
+
+            tokenName: root.tokenName
+            tokenImage: root.tokenImage
+            amount: root.tokenAmount === 0 ? "" : root.tokenAmount.toString()
```

Comment:
Maybe I'm blind but I don't see `root.tokenAmount` declared anywhere :)

I was about to remind you of `root.tokenAmount.toString()` being wrong... (ie not respecting the locale)

Reply:
It's declared, line 16 :)